src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset name="personalInformation">
+        <legend>
+          Personal information
+        </legend>
+
+        <label class="form_label">
+            Surname:
+          <input
```

Comment:
```suggestion
        <label class="form_label">
          Surname:
          <input
```

Reply:
Hello. I did the requested changes and double-checked them, but I am not sure if you have received them.